examples/einsum.py

```diff
@@ -112,6 +120,7 @@ def run_einsum(expr, N, iters, dtype):
 
 
 if __name__ == "__main__":
+    global cupy_compatibility
```

Comment:
Probably instead of using a `global`, we should pass this as an argument (like the other parameters). In general, we avoid using `global`.

Reply:
Fixed. Thanks.